### PR TITLE
Add Pronamic Pay meta box to WooCommerce admin order page

### PR DESCRIPTION
**Pull request by remcotolsma**
Scherm­afbeelding 2023-09-18 om 15 56 25

### CODE REVIEW

File: views/admin-meta-box-woocommerce-order.php [hunk 1]
```diff
@@ -0,0 +1,88 @@
...
+			</tr>
+			<tr>
+				<th scope="row">
+					<?php esc_html_e( 'Amount', 'pronamic_ideal' ); ?>
+				</th>
+				<td>
+					<?php echo esc_html( $payment->get_total_amount()->format_i18n() ); ?>
+				</td>
+			</tr>
+			<tr>
```
Comment (rvdsteege) [inline views/admin-meta-box-woocommerce-order.php:59]: Also include payment status in the meta box?

```suggestion
			<tr>
				<th scope="row">
					<?php esc_html_e( 'Status', 'pronamic_ideal' ); ?>
				</th>
				<td>
					<?php echo esc_html( $payment->get_status_label() ); ?>
				</td>
			</tr>
			<tr>
```

File: src/Extension.php [hunk 2]
```diff
@@ -129,6 +129,36 @@ public static function init() {
...
+						'woocommerce-order-pronamic-pay',
+						\__( 'Pronamic Pay', 'pronamic_ideal' ),
+						function( $post ) use ( $order ) {
+							include __DIR__ . '/../views/admin-meta-box-woocommerce-order.php';
+						},
+						$post_type,
+						'side',
+						'default'
+					);
+				},
```
Comment (rvdsteege) [inline src/Extension.php:156]: Move to an `Admin` class or at least a class method, instead of an anonymous function?

**Review by rvdsteege — CHANGES_REQUESTED** [review]

File: src/Extension.php (hunk 2 above)
Comment (remcotolsma) [reply src/Extension.php:156]: I thought an admin controller class was a bit too much for just this. A separate function could be possible, but does that have any advantages here?

Comment (rvdsteege) [reply src/Extension.php:156]: IMHO, the `init()` function is getting a bit cluttered with an anonymous function. Calling `add_action()` with a clear function name improves readability of the code.

**Review by rvdsteege — APPROVED** [review]

**Pull request merged**